spark_auto_mapper_fhir/backbone_elements/immunization_protocolApplied_backbone_element.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from spark_auto_mapper_fhir.complex_types.codeableConcept import CodeableConcept
+from spark_auto_mapper_fhir.complex_types.reference import Reference
+from spark_auto_mapper_fhir.fhir_types.id import FhirId
+from spark_auto_mapper_fhir.backbone_elements.fhir_backbone_element_base import (
+    FhirBackboneElementBase,
+)
+from spark_auto_mapper_fhir.fhir_types.positive_int import FhirPositiveInt
+from spark_auto_mapper_fhir.fhir_types.string import FhirString
+from spark_auto_mapper_fhir.resources.organization import Organization
+from spark_auto_mapper_fhir.valuesets.immunization_target_disease import (
+    ImmunizationTargetDiseaseCode,
+)
+
+
+class ImmunizationProtocolAppliedBackboneElement(FhirBackboneElementBase):
+    def __init__(
+        self,
+        *,
+        id_: Optional[FhirId] = None,
+        series: Optional[FhirString] = None,
+        authority: Optional[Reference[Organization]] = None,
+        targetDisease: Optional[CodeableConcept[ImmunizationTargetDiseaseCode]] = None,
+        doseNumberPositiveInt: FhirPositiveInt,
+        doseNumberString: FhirString,
```

Comment:
you may need to add "= None" at the end of these; I know doesNumber is a mandatory field, but the output can't have both

Reply:
Thank you. Added the "=None,"